code/modules/projectiles/projectile.dm

```diff
@@ -50,6 +50,7 @@
 	var/jitter = 0
 	var/embed = 0 // whether or not the projectile can embed itself in the mob
 	var/forcedodge = 0 //to pass through everything
+	var/silence

```

Comment:
Why does this exist, you don't even do anything with it. Furthermore why are you making a var that is meant to be a number, but is explicitly set to null?